File: PurchaseSale/Api/PurchaseSale.Entity/OperationLog.cs
using OpenDataAccess.Entity;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PurchaseSale.Entity
{
    [TableProperty(Name = "t_OperationLog", PrimaryKey="LogId")]
    [RequestMethod(IsDelete = false, IsPost = false, IsPut = false, IsGet = false)]
    public class OperationLog : EntityModel, IEntity
    {
        public Guid LogId { get; set; }

        public string LogType { get; set; }

        public string LogPath { get; set; }

        public string EntityName { get; set; }

        public string RequestType { get; set; }

        public string MethodName { get; set; }

        public string IPAddress { get; set; }

        public DateTime StartTime { get; set; }

        public DateTime EndTime { get; set; }

        public long ElapsedMilliseconds { get; set; }

        public string OperationUser { get; set; }

        public DateTime CreateDate { get; set; }
    }

    [TableProperty(Name = "v_OperationLog", PrimaryKey = "LogId")]
    [RequestMethod(IsDelete = false, IsPost = false, IsPut = false)]
    public class ViewOperationLog : OperationLog
    {
        public string UserName { get; set; }
        public string LookDetail { get; set; }
    }
}

File: StoryTestCases/Web/configs/entities/OperationLog.js
module.exports = {
    Name: "OperationLog",
    PrimaryKey: "LogId",
    Properties: GetProperties()
}

function GetProperties() {
    return [
        GetProperty("LogId", "LogId"),
        GetProperty("LogType", "类型"),
        GetProperty("RequestType", "请求类型"),
        GetProperty("EntityName", "实体名"),
        GetProperty("MethodName", "方法名"),
        GetProperty("CreateDate", "操作时间"),
        GetProperty("UserName", "操作人"),
        GetProperty("StartTime", "开始时间"),
        GetProperty("EndTime", "结束时间"),
        GetProperty("ElapsedMilliseconds", "耗时(毫秒)"),
        GetProperty("LookDetail", "详细"),
        GetProperty("IPAddress", "IP地址")
    ]
}

function GetProperty(Name, Label) { return { Name, Label } }